src/pages/home/report/ReportActionItem.js

```diff
@@ -69,6 +76,20 @@ class ReportActionItem extends Component {
         };
         this.checkIfContextMenuActive = this.checkIfContextMenuActive.bind(this);
         this.showPopover = this.showPopover.bind(this);
+        this.animatedBackgroundColor = new Animated.Value(0);
+        const interpolatedBackgroundColor = this.animatedBackgroundColor.interpolate(
+            {
+                inputRange: [0, 1],
+                outputRange: [colors.white, colors.honeydew],
+            },
+        );
+        this.animatedBackgroundStyle = {
+            backgroundColor: interpolatedBackgroundColor,
+        };
+    }
```

Comment:
NAB, but maybe this could be kept outside of the Component for better readability/reuse?

Reply:
Hmmm, i'm not sure how that would work. Did you meant to move `animatedBackgroundColor`, `interpolatedBackgroundColor` and `animatedBackgroundStyle` outside the component? I tried that and it doesn't work very well. I tried checking our code base and usually animations are within the component so let me know how you meant to achieve the change.